Sources/CryptoSwift/HMAC.swift

```diff
@@ -20,7 +20,7 @@ public final class HMAC: Authenticator {
   }
 
   public enum Variant {
-    case sha1, sha256, sha384, sha512, md5
+    case sha1, sha256, sha384, sha512, sha3_224, sha3_256, sha3_384, sha3_512, md5
```

Comment:
let's simplify the API a bit by using a parameter here, I'm thinking about reuse SHA3.Variant here like this:

```swift
case sha3(SHA3.Variant)
```